### PR TITLE
Overall readability improvements

### PR DESCRIPTION
**Pull request by jiskattema**

### CODE REVIEW

**Comment by LourensVeen** [comment]
Our new IP policy is at https://www.esciencecenter.nl/nlesc_ip_policy_2017.pdf, so maybe that link could be updated while we're at it?

File: software/licensing.md [hunk 1]
```diff
@@ -4,19 +4,32 @@
...
+Compatibility should be checked when new external code is added to the project.
 
-## NOTICE(.txt|.md)
+## LICENSE
 
-Listing licenses, request citation of paper if applicable.
+The actual license of the code is stored in the *LICENSE* file.
+Github can add this file automatically when you create a new repository, or you can [add it via the repositories Github page](https://help.github.com/articles/adding-a-license-to-a-repository/).
+
+## NOTICE
```
Comment (sverhoeven) [inline software/licensing.md]: According to Apache license the NOTICE should not only contain our attribution,  but should also contain the NOTICE text of any included dependencies which have a NOTICE.

File: software/licensing.md [hunk 2]
```diff
@@ -4,19 +4,32 @@
...
+
+NOTICE should contain only the following text, adapted with the product's name and copyright dates:
+```
+  [PRODUCT_NAME]
+  Copyright [XXXX-XXXX] The Netherlands eScience Center, [PROJECT_PARTNERS]
+
+  This product includes software developed at
+  The Netherlands eScience Center (https://www.netherlandsesciencecenter.nl/).
```
Comment (sverhoeven) [inline software/licensing.md]: Our homepage is https://www.esciencecenter.nl

Comment (jiskattema) [reply software/licensing.md]: thanks, fixed

File: software/licensing.md (hunk 1 above)
Comment (jiskattema) [reply software/licensing.md]: ok, added that

**Pull request merged**